packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -151,8 +151,9 @@ export class DefaultCrudRepository<T extends Entity, ID>
     return this.toEntities(models);
   }
 
-  async findOne(filter?: Filter, options?: Options): Promise<T> {
+  async findOne(filter?: Filter, options?: Options): Promise<T | null> {
     const model = await ensurePromise(this.modelClass.findOne(filter, options));
+    if (!model) return null;
```

Comment:
A nitpick: `if (model == null) return null;`

Reply:
@raymondfeng shall I fix it your nitpick ;)